pybamm/models/base_models.py

```diff
@@ -537,12 +569,12 @@ def __init__(self):
             },
         )
 
-        # Current
-        icell = pybamm.electrical_parameters.current_with_time
-        icell_dim = pybamm.electrical_parameters.dimensional_current_with_time
-        self.variables.update(
-            {"Total current density": icell, "Total current density [A m-2]": icell_dim}
-        )
+        # Overwrite geometry
+        self.default_geometry = pybamm.Geometry("1D macro")
+
+        # Standard time variable
+        time_scale = pybamm.standard_parameters_lead_acid.tau_discharge
+        self.variables.update({"Time [h]": pybamm.t * time_scale / 3600})
```

Comment:
Could we add time in seconds and time in minutes aswell? Also discharge capacity (if its simple to do here, otherwise will create another issue)

Reply:
Should be easy enough. Discharge capacity in [Ah]?